src/Sulu/Bundle/RouteBundle/Tests/Functional/Controller/RouteControllerTest.php

```diff
@@ -215,6 +215,31 @@ public function testGenerateObjectAccess(): void
         $this->assertSame('/prefix/2023/12/24/christmas-party', $result['resourcelocator']);
     }
 
+    public function testGenerateObjectImplodeAccess(): void
+    {
+        $this->client->jsonRequest(
+            'POST',
+            '/api/routes?action=generate',
+            [
+                'locale' => 'de',
+                'resourceKey' => 'event-resource-key',
+                'entityClass' => 'event-class',
+                'routeSchema' => '/prefix/{implode("-", object)}',
```

Comment:
@wachterjohannes we can send the `routeSchema` via API to the backend?

Reply:
yes we can 😂

Reply:
🙈